docs/tutorials/annotation/annotate_points.md

```diff
@@ -99,12 +99,12 @@ def point_annotator(
 
     viewer = napari.view_image(stack)
     points_layer = viewer.add_points(
-        properties={'label': labels},
+        feature_defaults={'label': labels},
```

Comment:
@jni I have changed to `feature_defaults` in the constructor now. 
However, when running this code it now gives the following error:
```
  File "/home/pape/Work/my_projects/napari/doc-example/annotation.py", line 38, in main
    points = v.add_points(
  File "/home/pape/Work/my_projects/napari/napari/napari/components/viewer_model.py", line 4, in add_points
    import itertools
  File "/home/pape/Work/my_projects/napari/napari/napari/layers/points/points.py", line 440, in __init__
    self._feature_table = _FeatureTable.from_layer(
  File "/home/pape/Work/my_projects/napari/napari/napari/layers/utils/layer_utils.py", line 882, in from_layer
    return cls(features, defaults=feature_defaults, num_data=num_data)
  File "/home/pape/Work/my_projects/napari/napari/napari/layers/utils/layer_utils.py", line 714, in __init__
    self._defaults = _validate_feature_defaults(defaults, self._values)
  File "/home/pape/Work/my_projects/napari/napari/napari/layers/utils/layer_utils.py", line 947, in _validate_feature_defaults
    raise ValueError(
ValueError: Feature defaults contain some extra columns not in feature values: {'label'}
```
So it seems that it's currently not possible to pass the `feature_defaults` without also passing `features`. But in this case we don't have `features` (or points) yet, since they will be provided through the user annotations.
This error happens with the napari from the latest commit on `main`.

Is this a general issue with the `features` functionality (in that case I am happy to create an issue in `napari`), or am I still understanding or doing something wrong?

cc @andy-sweet

Reply:
It looks like a bug in the current napari. You should open an issue or PR.

Reply:
See https://github.com/napari/napari/issues/5790

Reply:
This is fixed by passing empty features.